pgtrigger/installation.py

```diff
@@ -52,7 +59,12 @@ def prunable(database=None):
     }
 
     with utils.connection(database).cursor() as cursor:
-        pg_maj_version = int(str(cursor.connection.server_version)[:-4])
+        conn = cursor.connection
+        if HAVE_PSYCOPG3 and isinstance(conn, Psycopg3Connection):
+            version = conn.info.server_version
+        else:
+            version = conn.server_version
+        pg_maj_version = int(str(version)[:-4])
```

Comment:
Moved to `Connection.info` - https://www.psycopg.org/psycopg3/docs/api/objects.html#psycopg.ConnectionInfo.server_version